distributed/scheduler.py

```diff
@@ -8062,8 +8062,26 @@ def adaptive_target(self, target_duration=None):
 
         # CPU
 
-        # TODO consider any user-specified default task durations for queued tasks
-        queued_occupancy = len(self.queued) * self.UNKNOWN_TASK_DURATION
+        if len(self.queued) < 100:
+            queued_occupancy = 0
+            for ts in self.queued:
+                if ts.prefix.duration_average == -1:
+                    queued_occupancy += self.UNKNOWN_TASK_DURATION
+                else:
+                    queued_occupancy += ts.prefix.duration_average
```

Comment:
Out of scope nit: this screams for encapsulation in a smart duration_average property

Reply:
I don't disagree

Reply:
There is [`get_task_duration`](https://github.com/dask/distributed/blob/9eb672840cc76f8718227fc1d2973cc6f233b9cd/distributed/scheduler.py#L2978-L2995) which also handles the case of user provided estimates.